### ISSUE TITLE
React 15.0.0 support

### ISSUE THREAD

**Issue by Andreyco** [facebook/fixed-data-table#364]
Are you considering React 15.0.0 support?
Thx

**Comment by johannessjoberg** [comment]
+1

Repo still alive?

**Comment by 3cooper** [comment]
I have the same question as johannessjoberg.  I have submitted a few issues with JSFiddles and I haven't even heard if what I found is a bug or if I'm doing something the wrong way.  I understand the devs are very busy, and I appreciate all of Facebook's work on this but I'd like to know if this is something that will continue to evolve and be supported.

**Comment by ChristophBoget** [comment]
+1

I need to know about this as well.  Is this repo still alive?

**Comment by zthomas1** [comment]
Agreed, some kind of an update would be most appreciated.

**Comment by BartoszWu** [comment]
+1 
Please add update about repo.

**Comment by digifuzz** [comment]
+1 ditto

**Comment by 592da** [comment]
+1
Please keep updating this repo !!!

**Comment by bryce-larson** [comment]
+1
Please keep keep this repo up to date. 
Please at least gives as a update on the future of this repo

**Comment by Martynas-P** [comment]
I think you got your answer.

**Comment by Andreyco** [comment]
Maybe community should take responsibility for this.
At least, we find out whether FB is willing to review & merge potential fixes/addons

**Comment by amertak** [comment]
+1

**Comment by cpinnix** [comment]
Looks like support for this just dropped off? 5 months since the last commit. Such a shame. Does anyone know of a well supported comparable library?

**Comment by EvNaverniouk** [comment]
@cpinnix The closest thing I've seen so far is the FlexTable that ships with react-virtualized: https://bvaughn.github.io/react-virtualized/

**Comment by 3cooper** [comment]
Does anyone here have experience with FlexTable from react-virtualized?  I have been holding off trying it because this project has worked well, and I've been hoping that we'd get some sort of answer as to the future of this project.  This issue alone is 1 month old and we haven't heard anything as to whether or not this project is abandoned.

**Comment by cpinnix** [comment]
@EvNaverniouk Thanks for the heads up. Looks like the demo mobile performance is pretty good. Going to check react-virtualized when I get a chance. Probably sometime soon.

**Comment by jljorgenson18** [comment]
Unfortunately supporting IE9 makes it difficult to use a table built with flexbox. The only other table component I saw was the react boostrap table. Does anybody know of any other table components?

**Comment by rbalicki2** [comment]
Hey - does anyone know if this is actually broken with react 15?

I upgraded to react 15 on a branch, and it appears that the fixed-data-table continues to work. I don't use all of the features, of course.

**Comment by 3cooper** [comment]
@rbalicki2 - the fiddles I had were not for this issue.  I was mentioning the fiddles to show that it should have been easy for someone to look at an issue and say yes this is an issue or no you are doing it wrong.

In case you are still interested, the fiddles are in this issue - https://github.com/facebook/fixed-data-table/issues/326

**Comment by HillLiu** [comment]
+1

**Comment by HillLiu** [comment]
👍

**Comment by bytecodedan** [comment]
+1

**Comment by pongsakornsemsuwan** [comment]
+1 
Anyone have an alternate react table to recommend? just in case.

**Comment by johannessjoberg** [comment]
We have moved on to react-bootstrap-table (https://github.com/AllenFang/react-bootstrap-table).
Works very well!
I filed some improvement suggestions last week and got a response within half an hour. This week they were implemented :)

**Comment by 3cooper** [comment]
I'm looking for one that supports resizable columns and fixed header.  I also really like the virtual rows.

**Comment by swarajban** [comment]
If anyone is looking for a quick 'fix' to this, update your package.json dependency for fixed-data-table to:

```
"fixed-data-table": "jneiku/fixed-data-table#9617de14a14e2ece87590c6b118631755eff1fc8"
```

The [jneiku/fixed-data-table fork](https://github.com/jneiku/fixed-data-table) removes the <= 14 peer dependency, and has all the built files checked into the repo. Unlike the other forks that simply update the peer dependency, this is important for webpack requires certain built files to exist that are published in npm, but are not checked into the github repo.

FYU I'm using React 15 + webpack. Not seeing any warning / deprecation warnings, and my tables are functioning correctly.

Hope this helps people get by until either facebook properly updates this repo, or you switch to a better supported table library

**Comment by swarajban** [comment]
@cpinnix I'm going to move my tables over to the better supported [Griddle](https://github.com/GriddleGriddle/Griddle) react table library

**Comment by amertak** [comment]
I moved my table to react-virtualized just under 2 hours with all stylings and so. I find it much better.

**Comment by cpinnix** [comment]
@swarajban I've moved to [react-virtualized](https://github.com/bvaughn/react-virtualized) like @amertak. It's a bit tricky to dynamically update each cell as data changes after the initial render. But things work well not that everything is together.

**Comment by malykhinvi** [comment]
> Repo still alive?

@johannessjoberg seems like this particular repo is [abandoned](https://twitter.com/ryanseddon/status/692452531016986624).

**Comment by KamranAsif** [comment]
Hey folks,

My group is using this library very actively, and we plan to maintain it. 
We've already merged in some pull requests (Notably column drag reorder) and have done several bug fixes and have support for react v15.
We also threw up some new examples to help folks!

Check it out here: https://github.com/schrodinger/fixed-data-table

**Comment by zpao** [comment]
We shipped 15.0 support a couple weeks ago.
